src/persistence/settings.cpp

```diff
@@ -1379,6 +1409,48 @@ void Settings::setCircleExpanded(int id, bool expanded)
     circleLst[id].expanded = expanded;
 }
 
+void Settings::addFriendRequest(const QString &friendAddress, const QString &message)
+{
+    QMutexLocker locker{&bigLock};
+    QPair<QString, QString> request(friendAddress, message);
+
+    if (friendRequests.indexOf(request) != -1)
```

Comment:
@TheSpiritXIII Please add brackets `{ }` for better code readability and less bugs. :)